test/codegen/fromcto/plantuml/plantumlvisitor.js

```diff
@@ -44,14 +44,16 @@ describe('PlantUMLVisitor', function () {
         getFullyQualifiedTypeName: () => 'String',
         getName: () => 'name',
         isArray: () => true,
-        isTypeEnum: () => false
+        isTypeEnum: () => false,
```

Comment:
Let's add a case where the list of decorators isnt null

Reply:
> Let's add a case where the list of decorators isnt null

Since we are dropping the representation of decorators in the UML, I would add test to validate that they don't exist in the UML